packages/mesa.rb

```diff
@@ -27,8 +27,12 @@ class Mesa < Package
   depends_on 'libxkbfile'
   depends_on 'wayland'
 
+  depends_on 'bison'
```

Comment:
You will need `depends_on 'python27'` since you are using `pip install` below.

Reply:
yes